integrated_channels/integrated_channel/management/commands/transmit_courseware_data.py

```diff
@@ -78,4 +78,14 @@ def send_data_task(username, channel_code, channel_pk):
         channel,
     )
 
-    channel.transmit_course_data(user)
+    # channel.transmit_course_data(user)
+    try:
+        channel.transmit_course_data(user)
+    except Exception:  # pylint: disable=broad-except
+        exception_message = 'Transmission of course data failed for user "{username}" and for integrated channel ' \
+                        'with code "{channel_code}" and id "{channel_pk}".'.format(
+                            username=username,
+                            channel_code=channel_code,
+                            channel_pk=channel_pk,
+                        )
+        LOGGER.exception(exception_message)
```

Comment:
we need to also log information about the exception that was caught.

Reply:
LOGGER.exception will include traceback for exception.